blocks/library/block/index.js

```diff
@@ -41,6 +41,15 @@ class ReusableBlockEdit extends Component {
 		}
 	}
 
+	/**
+	 * @inheritdoc
```

Comment:
I guess this is required by the new JS Docs which is fine. Do we want to use `@inheritdoc` for all React lifecycle methods? Do you think we should add message explaining the logic for this specific case "exit edit mode when unfocusing"? Or maybe the code is too obvious?

Reply:
I think in this case that the code is pretty self explanatory. Personally, I'd prefer we do `@inheritdoc` for React lifecycle methods so that they don't distract a developer from the _real_ doc comments that have _real, useful_ information.